packetnetworking/distros/distro_builder.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, metadata):
         self.metadata = metadata
         self.network = self.metadata.network
         self.builders = []
-        self.tasks = None
+        self.tasks = {}
```

Comment:
This change breaks line 116
```python
if self.tasks is None:
  self.build()
```

Making sure we don't call build multiple times.